### PR TITLE
Run fieldalignment on all structs

### PR DESCRIPTION
**Pull request by adjackura**
https://pkg.go.dev/golang.org/x/tools@v0.1.0/go/analysis/passes/fieldalignment

### CODE REVIEW

File: agentendpoint/inventory.go [hunk 1]
```diff
@@ -28,6 +29,7 @@ func (c *Client) ReportInventory(ctx context.Context) {
 	state := inventory.Get(ctx)
 	write(ctx, state, inventoryURL)
 	c.report(ctx, state)
+	debug.FreeOSMemory()
```
Comment (ryanwe) [inline agentendpoint/inventory.go:32]: nit: can you mention this in the commit description too?

**Review by ryanwe — APPROVED** [review]
/hold

File: agentendpoint/inventory.go (hunk 1 above)
Comment (adjackura) [reply agentendpoint/inventory.go:32]: ugh nice catch, I meant to remove this, but this means my testing was not as useful as I thought :(

**Review by ryanwe — APPROVED** [review]
/unhold
/lgtm

**Pull request merged**